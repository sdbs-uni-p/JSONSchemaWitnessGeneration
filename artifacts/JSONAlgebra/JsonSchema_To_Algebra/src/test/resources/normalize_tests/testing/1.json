{
  "allOf": [
    {
      "const": 1
    },
    {
      "not": {
        "allOf": [
          {
            "$ref": "#foo"
          }
        ],
        "definitions": {
          "A": {
            "type": "integer",
            "$id": "#foo"
          }
        }
      }
    }
  ]
}