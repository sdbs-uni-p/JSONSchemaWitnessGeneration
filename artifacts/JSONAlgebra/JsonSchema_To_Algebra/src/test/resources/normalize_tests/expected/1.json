{"allOf":[{"const":1},{"not":{"allOf":[{"$ref":"#/definitions/id_foo"}],"definitions":{"A":{"type":"integer"}}}}],"definitions":{"id_foo":{"type":"integer"}}}